src/components/menu/menu-item/menu-item.component.js

```diff
@@ -95,8 +92,6 @@ MenuItem.propTypes = {
   submenuDirection: PropTypes.string,
   /** Is the menu item the currently selected item. */
   selected: PropTypes.bool,
-  /** (for submenus) renders with a divide between items. */
-  divide: PropTypes.bool,
```

Comment:
I think we should because if user was using the divider then it will gone because divide prop has been removed
So he will need to update his component and use `<MenuDivider />`

Reply:
great, I see it's in the commit message 👍